app/app.js

```diff
@@ -8,6 +9,41 @@ import config from "timed/config/environment";
 import "simplebar";
 import "simplebar/dist/simplebar.css";
 
+if (config["@sentry/ember"]) {
+  const sentryConfig = config["@sentry/ember"].sentry;
+  Sentry.init({
+    dsn: sentryConfig.dsn,
+    ignoreErrors: config.SENTRY_IGNORE,
+    tracesSampleRate: 1.0,
+    debug: sentryConfig.debug,
+    beforeSend(event, hint) {
+      // ignore this differentiation in dev environment
+      if (event.environment === "production") {
+        if (window.location.hostname === "test.timed.adfinis.com") {
+          event.environment = "staging";
+        }
+      }
+
+      const exception = hint.originalException;
+      if (
+        exception &&
+        Array.isArray(exception.errors) &&
+        exception.errors.every((e) => e.status === 401)
+      ) {
+        event.level = "info";
+      }
+
+      if (
+        config.SENTRY_IGNORE &&
+        config.SENTRY_IGNORE.includes(hint.originalException.name)
+      ) {
+        return null;
+      }
+      return event;
```

Comment:
I would like to see some filtering of [PII](https://www.ibm.com/topics/pii) relevant information before sending it to third party services. 

An example is given [here](https://docs.sentry.io/platforms/javascript/guides/ember/configuration/filtering/?original_referrer=https%3A%2F%2Fgithub.com%2Fgetsentry%2Fsentry-javascript%2Ftree%2Fmaster%2Fpackages%2Fember#filtering-error-events)

Reply:
You could implement the `event Object` filtering with [fast-redact](https://www.npmjs.com/package/fast-redact). Then all you need to do is look up the relevant keys in our models. :-)